modules/contact/src/problems/AugmentedLagrangianContactProblemInterface.C

```diff
@@ -0,0 +1,26 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "AugmentedLagrangianContactProblemInterface.h"
+
+InputParameters
+AugmentedLagrangianContactProblemInterface::validParams()
+{
+  auto params = emptyInputParameters();
+  params.addParam<int>("maximum_lagrangian_update_iterations",
+                       100,
+                       "Maximum number of update Lagrangian Multiplier iterations per step");
```

Comment:
It's not worth creating an interface just for this

it could go in AugmentedLagrangianContactConvergence, in a separate inputParams() routine if you need it available from the problem too

Reply:
nevermind it s used by a ton of objects

Reply:
this is not good practice. Means tons of objects can have different values for this parameter instead of retrieving it from a central place